server/grpc/api/mapping.go

```diff
@@ -1,21 +1,14 @@
 package api
 
 import (
-	"github.com/mesg-foundation/core/hash"
 	"github.com/mesg-foundation/core/protobuf/definition"
 	"github.com/mesg-foundation/core/service"
 )
 
 // FromProtoService converts a the protobuf definition to the internal service struct
 // TODO: should not be public. Need to move server/grpc/service.go to server/grpc/api/service.go
 func FromProtoService(s *definition.Service) (*service.Service, error) {
-	hash, err := hash.Decode(s.Hash)
-	if err != nil {
-		return nil, err
-	}
-
 	return &service.Service{
-		Hash:          hash,
```

Comment:
I remove the mapping of the hash here because it's only use in the api service.create that accept service that should NOT have any hash.

Reply:
`service.Create` should not take a definition in the first place. It should be definition minus hash and configuration image.